bubble-wrap-style.yaml

```diff
@@ -2239,19 +2307,12 @@ layers:
                         - [13, 1.50]
                         - [14, 2.0]
                     cull_from_tile: true
-                    visible: false
+                    visible: true
                     text:
                         offset: [0px, -0.5px]
                         repeat_distance: 200px
                         anchor: center
-                        text_source: |
-                            function() {
-                                if( feature.shield_text ) {
-                                    return feature.shield_text;
-                                } else {
-                                    return feature.ref;
-                                }
-                            }
+                        text_source: [shield_text, ref]
```

Comment:
Native scene syntax FTW!!! 👌

Reply:
So, it turns out `shield_text` != `ref`, and more specifically `shield_text.length()` != `ref.length()`.

It looks like you've accounted for that in the new sprite function (above), though, so this logic would always be coherent.